src/SignalR/clients/ts/signalr/src/ServerSentEventsTransport.ts

```diff
@@ -38,7 +38,7 @@ export class ServerSentEventsTransport implements ITransport {
 
         this._logger.log(LogLevel.Trace, "(SSE transport) Connecting.");
 
-        // set url before accessTokenFactory because this.url is only for send and we set the auth header instead of the query string for send
+        // set url before accessTokenFactory because this._url is only for send and we set the auth header instead of the query string for send
         this._url = url;
 
         if (this._accessTokenFactory) {
```

Comment:
Can we cache this too? What if we only called it for the first request and after 401s observed in `onerror`? Can the cached access token be shared with the `AccessTokenHttpClient`?

Reply:
Might be able to pass the access token from HttpConnection into the ctor, since we would have just called the factory for negotiate, no need to call it again in SSE.